client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/step-content.tsx

```diff
@@ -9,11 +13,47 @@ type StepContentProps = {
 	goToStep?: NavigationControls[ 'goToStep' ];
 };
 
-const StepContent = ( { siteSlug, submit, goNext, goToStep }: StepContentProps ) => (
-	<div className="launchpad__content">
-		<Sidebar siteSlug={ siteSlug } submit={ submit } goNext={ goNext } goToStep={ goToStep } />
-		<LaunchpadSitePreview siteSlug={ siteSlug } />
-	</div>
-);
+function sortByRegistrationDateCompareFunction(
+	firstDomainObject: ResponseDomain,
+	secondDomainObject: ResponseDomain
+) {
+	if ( firstDomainObject.registrationDate > secondDomainObject.registrationDate ) {
```

Comment:
`NB`: Totally an opinion, but we can probably clean up the names to make the function easier to read. Something like this:

```javascript
function sortByRegistrationDate(
	domainObjectA: ResponseDomain,
	domainObjectB: ResponseDomain
) {
	const registrationDateA = domainObjectA.registrationDate;
	const registrationDateB = domainObjectB.registrationDate;

	if ( registrationDateA > registrationDateB ) {
		return -1;
	} else if ( registrationDateA < registrationDateB ) {
		return 1;
	}
	return 1;
}
```

Reply:
I agree, updated the PR